MTM-62614 add release notes about raising Messaging Service alarms on common errors (#3861)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-2025-332-0-raise-messaging-service-alarms-on-error.md

---
date: 2025-10-03

> title: "Raising Messaging Service alarms on common errors"

Reviewer: Suggestion: rework a little for readability
```suggestion
title: "Alarms raised for common Messaging Service errors"
```

product_area: Platform services
change_type:
  - value: change-QHu1GdukP

>     label: Feature

Reviewer: Question: Is this a big enough change to be a "Feature"? I would probably have gone with "Improvement".

Author: Yeah, that's better word for it (for some reason I considered only Feature or Bugfix/Fix)

component:
  - value: component-2Yri1-l3n
    label: Messaging Service
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-62614
version: 2025.332.0
---

> From now on the Platform will raise a Messaging Service alarm when messages will fail to be processed (e.g. by Notification 2.0 feature) due to common issues. Those issues initially are about reaching backlog quota or topic limit, so issues where the customer should take the action to resolve it.

Reviewer 2: ```suggestion
With this change, the {{< product-c8y-iot >}} platform raises a Messaging Service alarm when messages fail to be processed (for example, by the Notification 2.0 feature) due to common issues. These issues initially are about reaching backlog quota or topic limit, so issues were the customer should take the action to resolve it.
```

> With this change, the {{< product-c8y-iot >}} platform raises a Messaging Service alarm when messages fail to be processed (for example, by the Notification 2.0 feature) due to common issues. These issues initially are about reaching backlog quota or topic limit, so issues were the customer should take the action to resolve it.

Reviewer: Suggestion: rework a little for readability, and don't restrict the set of things we might raise alarms for, as I expect we will want to extend this in future.
```suggestion
With this change, the {{< product-c8y-iot >}} platform will raise an alarm when the Messaging Service fails to process a message.
The issues reported in this way are those that need user action to resolve, for example when publishing to a Notifications 2.0 subscription fails because the underlying topic has reached its backlog quota limit.
```

---

> date: 2025-10-03

Reviewer 2: Please leave the date empty as it will be filled in automatically. Even if the change has already been deployed on eu-latest some time ago, it is not recommended to use an older date here, as people might not see the change log if they only check for the latest changes per date.

Author: Done!

title: "Alarms raised for common Messaging Service errors"
product_area: Platform services
change_type:
  - value: change-QHu1GdukP
    label: Improvement
component:
  - value: component-2Yri1-l3n
    label: Messaging Service
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-62614
version: 2025.332.0
---
With this change, the {{< product-c8y-iot >}} platform will raise an alarm when the Messaging Service fails to process a message.

> The issues reported in this way are those that need user action to resolve, for example when publishing to a Notifications 2.0 subscription fails because the underlying topic has reached its backlog quota limit.

Reviewer 2: ```suggestion
The issues reported in this way are those that need user action to resolve, for example, when publishing to a Notifications 2.0 subscription fails because the underlying topic has reached its backlog quota limit.
```

> The issues reported in this way are those that need user action to resolve. Currently, it will only happen when publishing to a Notifications 2.0 subscription fails because the underlying topic has reached its backlog quota limit. More information about Notification 2.0 backlog quotas can be found in the documentation: https://cumulocity.com/api/core/#section/Overview/Notification-2.0-Service-Quotas

Reviewer 2: ```suggestion
Issues reported in this way require user action to be resolved. Currently, this only happens when publishing to a Notifications 2.0 subscription fails because the underlying topic has reached its backlog quota limit. More information about Notification 2.0 backlog quotas can be found in the [documentation](https://cumulocity.com/api/core/#section/Overview/Notification-2.0-Service-Quotas).
```
